CHANGELOG.md

```diff
@@ -3,14 +3,6 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## v1.0.0 "Sweet potato" - [July 19th, 2024]
```

Comment:
You might also want to update the date here.

Reply:
Done :)